.github/workflows/ci.yml

```diff
@@ -49,13 +49,13 @@ jobs:
           node-version: [18, 20, 22]
           os: [ubuntu-latest, windows-latest, macos-latest]
           verification-script:
-            - pnpm --filter "\!*typescript*" build
-            - pnpm --filter "*typescript*" build
-            - pnpm --filter "*vitest*" test:unit
-            - pnpm --filter "*eslint*" lint --no-fix --max-warnings=0
-            - pnpm --filter "*prettier*" format --write --check
+            - pnpm --filter '!*typescript*' build
+            - pnpm --filter '*typescript*' build
+            - pnpm --filter '*vitest*' --filter '!*bare*' test:unit
```

Comment:
Does that mean the bare flag doesn't work with the vitest flag?

Reply:
It works with all flags, but the test scripts would fail initially, because:
- it removes all test files for vitest, so vitest would fail by "No test files found".
- it simplified App.vue drastically, so the html structures won't match the example e2e tests' expectations.

Reply:
I removed the example vitest specs because:
- they are used to test the components, but in the bare templates there isn't a `components` folder.
- the implementation is simpler with them removed.

I didn't touch the e2e specs because:
- I don't see people complaining about e2e specs being annoying
- I feel that people might still need some guidance on this. Because unlike unit tests, every e2e framework has different conventions on where to place the specs…

Reply:
But we don't usually unit test our `App.vue`? I'm afraid it's not what users would expect.

Reply:
Our teams do (we unit test all our components). I think it would be better to have a unit test, even if it is elementary. This is also the approach taken by the Angular CLI for example.

Reply:
I thought `App.vue` tests belong to E2E tests… Anyway, I've added it now

Reply:
❤️ Looks great!